### PR TITLE
Add terraform/helm setup script/instructions

### PR DESCRIPTION
**Pull request by AdamHawtin**
# Motivation and Context
We need the capability to run terraform/helm from cloudshell.

# What has changed
* Add terraform/helm setup script/instructions
* Fix a couple of mistakes

# How to test?
Run through the new instructions then try terraforming your dev environment from your cloudshell

# Links
https://trello.com/c/Qw8jaBZM/1386-document-script-terraform-helm-installation-on-cloudshell

### CODE REVIEW

**Comment by AdamHawtin** [comment]
[Document/script terraform/helm installation on cloudshell](https://trello.com/c/Qw8jaBZM/1386-document-script-terraform-helm-installation-on-cloudshell)

**Review by NickGrantONS — APPROVED** [review]
Nice documentation

File: README.md [hunk 1]
```diff
@@ -56,6 +56,33 @@ It is then important to:
 dewhitelist
 ```
 
+### Terraform and Helm
+We use terraform (via tfenv) and helm v2 with the tillerless plugin for our infrastructure management, these are the steps to set up those tools on your cloudshell. This is only necessary if you need to be able to make infrastructure changes.
+
+#### Creating a github personal token
+Clone the terraform repo with
```
Comment (liamtoozer) [inline README.md:63]: Remove this first line?

**Review by liamtoozer — APPROVED** [review]
LGTM 👍

**Pull request merged**